src/App.js

```diff
@@ -1,9 +1,9 @@
 import './App.css';
-import Calc from './components/Calc';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
@IjayAbby Good job changing the Calculator function to class-based but for some reason, you didn't change `App` to the class-based component as I mentioned in the previous review, kindly do that.

Example:

`class App extends Component {`